src/pages/Tools/Designer.js

```diff
@@ -48,11 +48,7 @@ const Designer = () => (
         framework. See some examples from Grommet designers below."
         color="DesignerYellow"
         images={['Designer_1.svg', 'Designer_2.svg', 'Designer_3.svg']}
-        labels={[
-          'Designer File Name',
-          'Designer File Name',
-          'Designer File Name',
-        ]}
+        labels={[null, null, null]}
```

Comment:
Can you just not send the prop and it will default to null?

Reply:
I tried that but this was the error I was getting 
<img width="1108" alt="Screen Shot 2020-08-05 at 2 13 03 PM" src="https://user-images.githubusercontent.com/49867133/89448591-dd03d880-d725-11ea-9e2a-a0889abef5f8.png">

Reply:
Probably because you are trying to approach label[i] while label is null.
Try to see if you can change it to `label={label && label.length ? label[i] : ""}`

Reply:
Updated to this in most recent push